testcases/OpTestLPM.py

```diff
@@ -209,24 +210,34 @@ def lpar_migrate_test(self):
             log.info("RMC service is inactive..!")
             self.rmc_service_start(self.src_mg_sys)
 
+        thread1 = OpSOLMonitorThread(1, "console")
+        thread1.start()
+
         cmd = ''
         if self.slot_num:
             cmd = self.vnic_options()
         if not self.cv_HMC.migrate_lpar(self.src_mg_sys, self.dest_mg_sys, self.options,
           cmd, timeout=self.lpm_timeout):
             self.lpm_failed_error(self.src_mg_sys)
 
+        thread1.console_terminate()
+
         if not self.is_RMCActive(self.dest_mg_sys):
             log.info("RMC service is inactive..!")
             self.rmc_service_start(self.dest_mg_sys)
 
+        thread2 = OpSOLMonitorThread(2, "console")
```

Comment:
here also variable name please

Reply:
@PraveenPenguin With above thread, We are not seeing any messages ..we  are not sure if the new mkvterm is being monitor or not here.